build.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file isn't necessary and can be deleted. It's a legacy file from the Ant-based plugin.